src/gerbil/test/c3-test.ss

```diff
@@ -16,7 +16,6 @@ prelude: "../prelude/core"
 (define gerbil/runtime 'gerbil/runtime)
 (define :gerbil/core ':gerbil/core)
 ;;(include "../../gerbil/runtime/util.ss")
-(include "../../gerbil/runtime/c3.ss")
```

Comment:
Comment it out rather than remove? We may need it at the next algorithm change.

Reply:
ok.

Reply:
done.